src/lib/components/Navbar.svelte

```diff
@@ -51,15 +50,21 @@ The navbar component.
         }
     }
 
+    $: actionBarColor = $globalConfig.themes
+        .find((x) => x.name === 'Normal' && x.enabled === true)
+        .colorSets.find((x) => x.type === 'main').colors['PrimaryColor'];
```

Comment:
I get the following error if "Normal" is not enabled:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'colorSets')
    at $$self.$$.update (Navbar.svelte:54:63)
    at init (index.mjs:1903:8)
    at new Navbar (Navbar.svelte:59:82)
    at createProxiedComponent (svelte-hooks.js?v=3062e50a:341:9)
    at new ProxyComponent (proxy.js?v=3062e50a:242:7)
    at new Proxy<Navbar> (proxy.js?v=3062e50a:349:11)
    at Array.create_default_slot (+layout.svelte:51:14)
    at create_slot (index.mjs:69:27)
    at create_fragment (Sidebar.svelte:48:67)
    at init (index.mjs:1907:37)
```

Until theme selection is implemented, I think I would leave off the ` && x.enabled == true`.

I am working on extending the globalConfig to have functions for looking up a style (e.g. ui.bar.action) and then resolving color values (e.g. ActionBarColor).

Reply:
Removed.